### PR TITLE
[Merged by Bors] - Fix clap for CI

### PR DESCRIPTION
**Pull request by hymm**
# Objective

- Fix CI
- relevant clap issue https://github.com/clap-rs/clap/issues/3822

## Solution

- slap `value_parser` on all the clap derives. This tells clap to use the default parser for the type.

### CODE REVIEW

**Comment by hymm** [comment]
I didn't have time yet to verify that this doesn't change any of the behavior of the tools.

**Review by alice-i-cecile — APPROVED** [review]
This matches the migration advice given, and fixes CI. You have my LGTM.

**Review by mockersf — CHANGES_REQUESTED** [review]
I think the expected way to fix it is to add `action` instead of `value_parser` everywhere

**Review by mockersf — APPROVED** [review]

**Comment by mockersf** [comment]
but I don't think it matters enough to block on it

**Pull request closed without merging**
